resources/install.rb

```diff
@@ -60,7 +60,7 @@
   remote_file "apache #{new_resource.version} tarball" do
     source tarball_uri
     path "#{Chef::Config['file_cache_path']}/apache-tomcat-#{new_resource.version}.tar.gz"
-    verify { |file| validate_checksum(file) }
+    verify { |file| validate_checksum(file) } if new_resource.verify_checksum
```

Comment:
The point of this one was to allow someone to self host on their own web server w/o the checksuming from apache.org. Baby steps